project_1948.py

```diff
@@ -300,33 +292,8 @@ def extract_interviewee(text_all, extracting_string, terminating_string):
     txt_combined = txt_combined.replace(',', '')
     # All the text is converted to its lower case
     txt_combined = txt_combined.lower()  
-    return txt_all, txt_combined 
-
-
-# The text is being cleaned and preprocessed before the topic model is built
-def clean_and_prep(text_all):
-    stop_free = " ".join([i for i in text_all.lower() if i not in stop and i not in ['interviewer', 'interviewee']])
-    punc_free = ''.join(ch for ch in stop_free if ch not in exclude)
-    normalized = " ".join(lemma.lemmatize(word) for word in punc_free.split())
-    return normalized
-
-# The topic model is being built below
-# The text corpus is inputted and the topic model is returned
-def build_topic_model(text_all, num_topics, passes):
-    # Creating the term dictionary of our corpus, where every unique term is assigned an index
-    dictionary = corpora.Dictionary([doc.split() for doc in text_all])
+    return txt_all, txt_combined
```

Comment:
Some functions in the original file like clean_and_prep, clean_text and words_in_sentences were removed. Can you please add them in your file too?